.github/workflows/unittest.yml

```diff
@@ -14,8 +16,7 @@ jobs:
         python-version: 3.7
     - name: Install Dependencies
       run: |
-        pip install netkan/.[test]
+        pip install --user netkan/. --no-warn-script-location --force-reinstall
     - name: Run unittest
```

Comment:
See previous comment.

Reply:
We actually don't use any dependencies for `unittest` - we can adjust that later if that changes.